docs/data/material/components/dividers/IntroDivider.js

```diff
@@ -23,7 +24,7 @@ export default function IntroDivider() {
           just down the hall.
         </Typography>
       </Box>
-      <Divider light />
+      <Divider sx={{ bgcolor: grey[50] }} />
```

Comment:
If we're not supporting the `light` prop anymore, I'd say it's okay to have all of these demos without the `sx` props. Way simpler to just use the color that comes out of the box!

Reply:
I agree 👍